src/rpc/Errors.h

```diff
@@ -17,6 +17,7 @@ namespace RPC {
  */
 enum class ClioError {
     rpcMALFORMED_CURRENCY = 5000,
+    rpcMARKER_NOT_FOUND = 5001,
```

Comment:
I'm now thinking whether we should use some rippled code instead of adding our own for this hack.. maybe `rpcOBJECT_NOT_FOUND`?